CHANGELOG.md

```diff
@@ -33,7 +32,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Fixes ingestPdrWithNodeNameSpec parsePdr provider error
 
 ### BREAKING CHANGES
-
+- **CUMULUS-2138**
+  - Adds a new task `update-granules-cmr-metadata-file-links` to Ingest and Publish Granule workflow. This new task updates CMR Metadata and adds CMR file etags. It removes said functionality from the `move-granules` task. It is recommended that you add the step `UpdateGranulesCmrMetadataFileLinksStep` to your workflows after `MoveGranulesStep`.
```

Comment:
I don't think it is recommended - it's required if you want these workflows to function as they did before. 

Let's move the note about the update under the "Migration notes" above with a note like:

```
- **CUMULUS-2138** - CMR metadata update behavior has been removed from the `move-granules` task into a 
new `update-granules-cmr-metadata-file-links` task. As a result, you need to update all workflows using the
`MoveGranules` step to add `UpdateGranulesCmrMetadataFileLinksStep`that runs after it. See the example
[`IngestAndPublishWorkflow`](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/ingest_and_publish_granule_workflow.asl.json) for reference.
```